### DISCUSSION TITLE
Add xsd (XML Schema) generator

### DISCUSSION THREAD

**Discussion by cmungall** [linkml/linkml#1932] [Archived]
indirectly, from a question from @johanneswerner to NMDC

Are there plans to write an xml-schema adapter?

I don't think this is in the critical path for the current linkml plans, but we would welcome any community contribution here

**Comment by cmungall** [comment]
one possibility is linkml->jsonschema->xsd, but the latter step does not seem well supported:

https://stackoverflow.com/questions/48376526/generate-xml-schemaxsd-from-json-schema

**Reply by l00mi** [reply]
I like to ask if there was any more discussions or work done in the direction to support also XSD as a generator?

We plan to use LinkML in a Standardisation body, and the path to also generate at least XSD, and better also translate from other Data representations to and from XML itself would be an important part.

Currently we see if we can contribute the necessary work, and therefor we like to know what the current status is.

**Reply by l00mi** [reply]
@cmungall we could secure some minor funding to push this forward. What are the good contacts to engage to get this started?

**Reply by matentzn** [reply]
@l00mi Nice to meet you! I am very happy to serve as you LinkML liaison on this - I recently joined the core team to coordinate its development activities. How can I help?
